pkgs/os-specific/linux/fxload/default.nix

```
@@ -1,22 +1,15 @@
-{lib, stdenv, fetchurl}:
+{lib, stdenv, fetchgit}:
 
 stdenv.mkDerivation rec {
   pname = "fxload";
-  version = "2002.04.11";
+  version = "2013-01-03";
```

Please update the commit message as well.

```suggestion
  version = "unstable-2013-01-03";
```

---

Update it to what?

---

Never mind, forgot this wasn't a package addition.

---

a'ight!